report_generator.gemspec

```diff
@@ -35,20 +35,20 @@ Gem::Specification.new do |spec|
 
   current_ruby_version = Gem::Version.new(RUBY_VERSION)
 
-  spec.add_dependency 'activerecord', '>= 4.2', '< 6'
-  spec.add_dependency 'activesupport', '>= 4.2', '< 6'
+  spec.add_dependency 'activerecord', '>= 4.2', '~> 6.1'
+  spec.add_dependency 'activesupport', '>= 4.2', '~> 6.1'
+  spec.add_dependency 'csv-safe', '>= 3.2.1' if Gem::Requirement.new('>= 2.6').satisfied_by?(current_ruby_version)
   spec.add_dependency 'dragonfly', '~> 1'
+  spec.add_dependency 'jwt', '~> 1.5'
   spec.add_dependency 'rails-html-sanitizer', '~> 1'
-  spec.add_dependency 'railties', '>= 4.2', '< 6'
+  spec.add_dependency 'railties', '>= 4.2', '~> 6.1'
```

Comment:
*(same comment as the one I made on csv2db)*

The `~> 6.1` constraint means that we can't use Rails 5.2 or 6.0 with the gem:

```ruby
# Latest Rails versions from https://endoflife.date/rails
Gem::Requirement.new(['>= 4.2', '~> 6.1']).satisfied_by? Gem::Version.new('5.2.8.1') #=> false
Gem::Requirement.new(['>= 4.2', '~> 6.1']).satisfied_by? Gem::Version.new('6.0.6.1') #=> false
```

since `~> 6.1` means "at least 6.1.whatever, or 6.2.x, 6.3.x etc, all the way up to (but not including) 7"